app/presenters/document_presenter.rb

```diff
@@ -41,7 +59,8 @@ def organisations
   end
 
   def extra_date_metadata
-    {}
+    finder.facets.select{ |f| f.type == 'date' }
+          .each_with_object({}) { |facet, hash| hash[facet.name] = self.try(facet.key) }
```

Comment:
I think this could just be `self.send(facet.key)` instead of `self.try(facet.key)`